client/src/components/popovers/QueueDisplay.vue

```diff
@@ -80,6 +84,9 @@ export default {
           eventCategory: 'playlistInteraction',
           eventAction: 'addSongToPlaylist'
         })
+      },
+      addSongToSpotifyQueue(uri){
+        this.addSongToQueue(uri)
```

Comment:
`this.addSongToQueue` doesn't exist. You're mapping the following action: `addSongToSpotifyQueue` and I guess it gets overridden by the local implementation?
I got it to work locally by renaming the local method (the one that's called by the button-press in the html).
And then changing `this.addSongToQueue` to `this.addSongToSpotifyQueue`

Reply:
Alternatively you could rename the mutation, but that would probably require more changes